internal/core/application/types.go

```diff
@@ -256,10 +258,13 @@ func (t *TimeRange) getStartAndEndTime(now time.Time) (startTime time.Time, endT
 			start = time.Date(y, time.January, 1, 0, 0, 0, 0, time.UTC)
 		case All:
 			start = time.Date(StartYear, time.January, 1, 0, 0, 0, 0, time.UTC)
+		case LastYear:
+			lastYear := now.Year() - 1
+			start = time.Date(lastYear, time.January, 1, 0, 0, 0, 0, time.UTC)
+			endTime = time.Date(lastYear, time.December, 31, 23, 59, 59, 0, time.UTC)
```

Comment:
Last year should be the range `[nowMinusOneYear, now]`:

```suggestion
			start = now.AddDate(-1, 0, 0)
```